content/minfraud/alerts.md

```diff
@@ -40,24 +40,71 @@ http://yourdomain/yoururl?i=24.24.24.24&maxmindID=1234ABCD&domain=sample.com&cit
+
+  {{< minfraud-schema-row key="date" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
+  The date of the original minFraud request, e.g., `Nov. 1, 2019`.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="domain" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
+  The email domain included in the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="i" type="webhook" valueType="string" valueTypeNote="format: IPv4 or IPv6" >}}
+  The IP address included in the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="maxmindID" type="webhook" valueType="string" valueTypeNote="max length: 8" >}}
+  The minFraud Legacy `maxmindID` of the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="minfraud_id" type="webhook" valueType="string" valueTypeNote="format: UUID" >}}
+  The minFraud ID of the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="new_risk_score" type="webhook" valueType="decimal" valueTypeNote="min: 0.01, max: 99" >}}
+  The updated risk score, calculated after your initial query with additional information.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="old_risk_score" type="webhook" valueType="decimal" valueTypeNote="min: 0.01, max: 99" >}}
+  The risk score as originally calculated.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="postal" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
+  The billing postal included in the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="reason" type="webhook" valueType="string" >}}
+  A human-readable description of why the minFraud alert was sent. See below for the current list of possible reasons.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="reason_code" type="webhook" valueType="string" valueTypeNote="format: enum" >}}
+  A fixed, machine-readable code for the reason why the minFraud alert was sent. See below for the current list of possible reasons.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="region" type="webhook" valueType="string" valueTypeNote="max length: 4" >}}
+  The billing region included in the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="shop_id" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
+  The shop ID included in the original minFraud request. This will only be set if the original request included a shop ID.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="txnID" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
+  The transaction ID included in the original minFraud request.
+  {{</minfraud-schema-row>}}
+
+  {{< minfraud-schema-row key="updated_at" type="webhook" valueType="string" valueTypeNote="max length: 255" >}}
```

Comment:
I'm not sure if we would want to show a max length for this. [This time field](https://dev.maxmind.com/minfraud/api-documentation/requests/#schema--request--event__time) does not list such a max length for instance

Reply:
@ugexe this one does https://dev.maxmind.com/minfraud/api-documentation/responses/#schema--response--ip-address--location__local_time.

@oschwald that PR was closed 🤔.

Reply:
A new version of it was opened. https://github.com/maxmind/dev-site/pull/1414